extmod/modujson.c

```diff
@@ -67,7 +67,7 @@ STATIC mp_obj_t mod_ujson_dump_helper(size_t n_args, const mp_obj_t *pos_args, m
         vstr_t vstr;
         vstr_init_print(&vstr, 8, &print_ext.base);
         mp_obj_print_helper(&print_ext.base, pos_args[0], PRINT_JSON);
-        return mp_obj_new_str_from_vstr(&mp_type_str, &vstr);
+        return mp_obj_new_str_from_vstr(&vstr);
```

Comment:
Does this need to validate utf8?  I would think that it's all already been validated as part of the dump process, or at least the dump should never produce invalid utf8.